README.md

```diff
@@ -70,13 +70,17 @@ public void ConfigureServices(IServiceCollection services)
     services.AddCap(x =>
     {
         // If you are using EF, you need to add the configuration：
-        x.UseEntityFramework<AppDbContext>(); //Options, Notice: You don't need to config x.UseSqlServer(""") again! CAP can autodiscovery.
+        x.UseEntityFramework<AppDbContext>(); //Options, Notice: Normally you don't need to configure x.UseSqlServer("") again! CAP can pick up the connection string from the EF context.
 
         // If you are using ADO.NET, choose to add configuration you needed：
         x.UseSqlServer("Your ConnectionStrings");
         x.UseMySql("Your ConnectionStrings");
         x.UsePostgreSql("Your ConnectionStrings");
 
+        // If you are using an Npgsql data source (to set password provider etc.), tell CAP to use it like this:
+        var dataSource = new NpgsqlDataSourceBuilder("Your ConnectionStrings").Build();
+        x.UsePostgreSql(options => options.DataSource = dataSource);
```

Comment:
I added an example here to show how it would be used. Let me know if you think it's too much detail to include here.

Reply:
I think the special instructions for postgresql can be explained in our docs and keep the readme concise.

Docs here:
https://github.com/dotnetcore/CAP/blob/master/docs/content/user-guide/en/storage/postgresql.md

Reply:
Okay, reverted this and added the property in the the storage docs instead.